packages/desktop/bin-src/dump_artifacts/index.ts

```diff
@@ -148,7 +150,10 @@ async function installPackages(
   requirements.push(stliteLibWheel);
   const streamlitWheel = await prepareLocalWheel(
     pyodide,
-    path.join(wheelsDir, "streamlit-1.41.0-cp312-none-any.whl"),
+    path.join(
+      wheelsDir,
+      `streamlit-${process.env.STREAMLIT_VERSION || "1.41.0"}-cp312-none-any.whl`,
```

Comment:
What provides the `STREAMLIT_VERSION` environment variable?
Looks like it's now working just because of the fallback value. Delete the fallback value for the "fail early" principle.